builds/e2e/templates/e2e-setup.yaml

```diff
@@ -37,6 +37,9 @@ steps:
 
     Write-Output "##vso[task.setvariable variable=imageBuildId]$imageBuildId"
     Write-Output "##vso[task.setvariable variable=packageBuildId]$packageBuildId"
+
+    Write-Output '>> Package Build ID=$(az.pipeline.packages.buildId)'
+    Write-Output '>> Image Build ID=$(az.pipeline.packages.buildId)'
```

Comment:
az.pipeline.images.buildId?

Reply:
That's a pipeline variable for the build image pipeline. ( a custom defined var)